scripts/prebuild.js

```diff
@@ -36,6 +36,12 @@ if (!process.env.GITHUB_ACTIONS) {
 }
 
 let commit_files = [];
+
+if (commits.length === 0) {
+  logger.log(`Didn't find any new commit. Exiting process.`);
+  return process.exit(1);
```

Comment:
Want to exit with 0? Since it is not actually an error or do we want to treat this as error, I guess not :)

Reply:
We have to use `exit(0)` because `npm run build:release` actually include these steps:
1. prebuild
2. build
3. postbuild

So if we exit(0) in step 1, step 2 and 3 will still be executed. So have to use `exit(1` to terminate the process.

Reply:
The difference between `npm run build` and `npm run build:release` is that the latter has `prebuild` script to check if there’s a `CHANGELOG` change included in the commit and `postbuild` scripts checks if there’s any uncommitted files after running test.
So if there’s no commit, it doesn’t make sense to run this script. We can manually run `npm run build` and `npm run test` instead.

Reply:
Yup makes sense can go ahead and merge.